clippy_utils/src/lib.rs

```diff
@@ -56,6 +56,81 @@ pub mod ty;
 pub mod usage;
 pub mod visitors;
 
+pub mod _internal {
```

Comment:
Why the underscore? I would've thought the name makes it clear that this is not for public consumption. Perhaps add `#[doc(hidden)]` instead?

Reply:
It's temporary anyways. Everything used will be regular `pub` items once I'm done.

Reply:
Ah, Ok then.